go-controller/pkg/cni/ovs.go

```diff
@@ -62,7 +55,10 @@ func ovsFind(table, column, condition string) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	return strings.Split(strings.TrimSpace(output), "\n"), nil
+	if output == "" {
+		return nil, nil
```

Comment:
(I think this is the only part of the patch that's actually having an effect, although I agree with the other `ovsExec` and `ovsFind` cleanups anyway.)